proto/stride/stakeibc/tx.proto

```diff
@@ -366,3 +369,18 @@ message MsgToggleTradeController {
   string address = 4;
 }
 message MsgToggleTradeControllerResponse {}
+
+// Updates host zone params
+message MsgUpdateHostZoneParams {
```

Comment:
Re:your comment, I'm on-board with adding a gov-callable function that can update host zone params!